index.html

```diff
@@ -3,9 +3,65 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Stack Over Flow</title>
 </head>
 <body>
+    <main>
+        <div class="content--left">
+          
+            <h1>Join the Stack Overflow community</h1>
+            <ul>
+              <li>Get unstuck — ask a question</li>
+              <li>Unlock new privileges like voting and commenting</li>
+              <li>Save your favorite questions, answers, watch tags, and more</li>
+              <li>Earn reputation and badges</li>
+            </ul>
+    
+            <p>Collaborate and share knowledge with a private group for FREE.</p>
+    
+            <a href="#">Get Stack Overflow for Teams free for up to 50 users.</a>
+          
+        </div>
+    
+      <div class="content--right">
+    
+        <div class="button-signup">
+    
+        <div class="google"><a href="https://myaccount.google.com/"><img src="logo-google.svg" alt="Google Logo" width="20" height="20">Sign up with Google</a></div>
```

Comment:
There's no need to wrap <a> elements inside divs. 
```suggestion
       						<a class="btn google" href="https://myaccount.google.com/"
							><img
								src="logo-google.svg"
								alt="Google Logo"
								width="20"
								height="20"
							/>Sign up with Google</a
						>
```

Apply to all 3 buttons here